prov/rxm/src/rxm_cq.c

```diff
@@ -607,39 +607,53 @@ void rxm_cq_write_error(struct util_cq *cq, struct util_cntr *cntr,
 		assert(0);
 }
 
-static size_t rxm_cq_read_write_error(struct fid_cq *msg_cq,
-				      struct fi_cq_data_entry *comp,
-				      ssize_t err)
+static void rxm_cq_write_error_all(struct rxm_ep *rxm_ep, int err)
+{
+	struct fi_cq_err_entry err_entry = {0};
+	ssize_t ret = 0;
+
+	err_entry.prov_errno = err;
+	err_entry.err = err;
+	if (rxm_ep->util_ep.tx_cq) {
+		ret = ofi_cq_write_error(rxm_ep->util_ep.tx_cq, &err_entry);
+		if (ret)
+			assert(0);
+	}
+	if (rxm_ep->util_ep.rx_cq) {
+		ret = ofi_cq_write_error(rxm_ep->util_ep.rx_cq, &err_entry);
+		if (ret)
+			assert(0);
```

Comment:
add log messages on cq write error failures.  The commit message should be updated to reflect what's actually happening -- we're getting an error trying to read a cq error and have no idea what else to do.

Reply:
👍